.github/workflows/quest-all.yml

```diff
@@ -0,0 +1,48 @@
+name: "bulk quest import"
```

Comment:
@IEvangelist This is a copy-paste-modify from the `quest-bulk.yml` workflow.

If there's a way to integrate this into the regular `bulk` workflow, I want your suggestions.

Reply:
What's different between the two?

Reply:
There are two differences:

The monthly version uses a duration of "-1". That instructs the code to walk all open issues with the configured labels. It also runs only once a month.

Is there a way to combine that so the schedule sets the parameters differently? E.g. every night, run with duration = 5, and once a month run with duration = -1?